.golangci.yml

```diff
@@ -86,9 +84,8 @@ linters-settings:
     # enable or disable analyzers by name
     enable:
       - atomicalign
+      - shaddow
     enable-all: false
-    disable:
-      - shadow
     disable-all: false
   golint:
```

Comment:
The `golint` linter was removed, so it should be removed from settings. Same with `maligned`